lib/client/db/postgres/servicefile.go

```diff
@@ -240,36 +174,17 @@ func (s *ServiceFile) Delete(name string) error {
 	return s.iniFile.SaveTo(s.path)
 }
 
-// ConnectProfile represents a single connection profile in the service file.
-type ConnectProfile struct {
-	// Name is the profile name.
-	Name string
-	// Host is the host to connect to.
-	Host string
-	// Port is the port number to connect to.
-	Port int
-	// User is an optional database user name.
-	User string
-	// Database is an optional database name.
-	Database string
-	// SSLMode is the SSL connection mode.
-	SSLMode string
-	// SSLRootCert is the CA certificate path.
-	SSLRootCert string
-	// SSLCert is the client certificate path.
-	SSLCert string
-	// SSLKey is the client key path.
-	SSLKey string
-}
-
-// pgServiceFile is the default name of the Postgres service file.
-const pgServiceFile = ".pg_service.conf"
-
-// SSLModeVerifyFull is the Postgres SSL "verify-full" mode.
-const SSLModeVerifyFull = "verify-full"
+const (
+	// SSLModeVerifyFull is the Postgres SSL "verify-full" mode.
```

Comment:
Can you link to the Postgres docs here - these comments are not super helpful in isolation.

Reply:
Done.